src/main/java/org/assertj/core/internal/Classes.java

```diff
@@ -231,6 +263,115 @@ public void assertHasDeclaredFields(AssertionInfo info, Class<?> actual, String.
     return fieldsName;
   }
 
+  /**
+   * Verifies that the actual {@code Class} has the {@code methods}.
+   *
+   * @param info contains information about the assertion.
+   * @param actual the "actual" {@code Class}.
+   * @param methods the methods who must be present in the class.
+   * @throws AssertionError if {@code actual} is {@code null}.
+   * @throws AssertionError if the actual {@code Class} doesn't contains all of the methods.
+   */
+  public void assertHasMethods(AssertionInfo info, Class<?> actual, String... methods) {
+    assertNotNull(info, actual);
+    doAssertHasMethods(info,actual, actual.getMethods(), methods);
+  }
+
+  /**
+   * Verifies that the actual {@code Class} has the declared {@code methods}.
+   *
+   * @param info contains information about the assertion.
+   * @param actual the "actual" {@code Class}.
+   * @param methods the methods who must be declared in the class.
+   * @throws AssertionError if {@code actual} is {@code null}.
+   * @throws AssertionError if the actual {@code Class} doesn't contains all of the methods.
+   */
+  public void assertHasDeclaredMethods(AssertionInfo info, Class<?> actual, String... methods) {
+    assertNotNull(info, actual);
+    doAssertHasMethods(info, actual, actual.getDeclaredMethods(), methods);
```

Comment:
thx for pointing that out. You are right, this one was still missing but should be fixed now.